apmpackage/apm/data_stream/service_interval_metrics/manifest.yml

```diff
@@ -0,0 +1,37 @@
+title: APM service metrics {{ .Interval }}
```

Comment:
Can this be processed by the `apmpackage/genpackage` command? I don't see those changes in the PR.